openquake/qa_tests_data/event_based_risk/case_master/job.ini

```diff
@@ -0,0 +1,70 @@
+[general]
+description = event based risk
+calculation_mode = event_based_risk
+
+[exposure]
+exposure_file = exposure_model.xml
+
+[boundaries]
+region_constraint = -122.6 38.3, -121.5 38.3, -121.5 37.9, -122.6 37.9
+
+[site_params]
+reference_vs30_type = measured
+reference_vs30_value = 760.0
+reference_depth_to_2pt5km_per_sec = 5.0
+reference_depth_to_1pt0km_per_sec = 100.0
+
+[erf]
+width_of_mfd_bin = 0.1
+rupture_mesh_spacing = 2.0
+area_source_discretization = 10
+
+[logic_trees]
+source_model_logic_tree_file = source_model_logic_tree.xml
+gsim_logic_tree_file = gsim_logic_tree.xml
+
+[correlation]
+ground_motion_correlation_model = JB2009
+ground_motion_correlation_params = {"vs30_clustering": True}
+
+[hazard_calculation]
+#intensity_measure_types = PGA, SA(0.1), SA(0.3), SA(0.4)
+random_seed = 24
+truncation_level = 3
+maximum_distance = 200.0
+investigation_time = 50
+number_of_logic_tree_samples = 0
+ses_per_logic_tree_path = 2
+
+[hazard_outputs]
+ground_motion_fields = true
+hazard_curves_from_gmfs = true
+mean_hazard_curves = true
+quantile_hazard_curves = 0.15, 0.50, 0.85
+hazard_maps = true
+poes = 0.02, 0.10
+uniform_hazard_spectra = true
+
+[vulnerability]
+structural_vulnerability_file = structural_vulnerability_model.xml
+nonstructural_vulnerability_file = nonstructural_vulnerability_model.xml
+contents_vulnerability_file = contents_vulnerability_model.xml
+business_interruption_vulnerability_file = downtime_vulnerability_model.xml
+occupants_vulnerability_file = occupants_vulnerability_model.xml
+
+[risk_calculation]
+risk_investigation_time = 50
+asset_correlation = 0.0
+loss_curve_resolution = 10
+loss_ratios = {"structural": [0.00010, 0.00016, 0.00025, 0.00040, 0.00065, 0.0010, 0.0016, 0.0025, 0.0040, 0.0065, 0.010, 0.016, 0.025, 0.040, 0.065, 0.10, 0.16, 0.25, 0.40, 0.65, 1.0], "nonstructural": [0.00010, 0.00016, 0.00025, 0.00040, 0.00065, 0.0010, 0.0016, 0.0025, 0.0040, 0.0065, 0.010, 0.016, 0.025, 0.040, 0.065, 0.10, 0.16, 0.25, 0.40, 0.65, 1.0], "contents": [0.00010, 0.00016, 0.00025, 0.00040, 0.00065, 0.0010, 0.0016, 0.0025, 0.0040, 0.0065, 0.010, 0.016, 0.025, 0.040, 0.065, 0.10, 0.16, 0.25, 0.40, 0.65, 1.0], "business_interruption": [0.00010, 0.00016, 0.00025, 0.00040, 0.00065, 0.0010, 0.0016, 0.0025, 0.0040, 0.0065, 0.010, 0.016, 0.025, 0.040, 0.065, 0.10, 0.16, 0.25, 0.40, 0.65, 1.0], "occupants": [0.00010, 0.00016, 0.00025, 0.00040, 0.00065, 0.0010, 0.0016, 0.0025, 0.0040, 0.0065, 0.010, 0.016, 0.025, 0.040, 0.065, 0.10, 0.16, 0.25, 0.40, 0.65, 1.0]}
+
+[risk_outputs]
+avg_losses = true
+insured_losses = true
+asset_loss_table = true
+quantile_loss_curves = 0.15, 0.50, 0.85
+conditional_loss_poes = 0.02, 0.10
+statistics = true
```

Comment:
This line can be removed in view of https://github.com/gem/oq-risklib/issues/630